gframe/settings_window.h

```diff
@@ -40,6 +40,7 @@ struct SettingsPane {
 	irr::gui::IGUICheckBox* chkNoChainDelay;
 
 	irr::gui::IGUICheckBox* chkIgnoreDeckContents;
+	irr::gui::IGUICheckBox* chkAddCardNamesInDeckList;
```

Comment:
The entry is not needed here since it's only shown in the settings window, this is the list of elements in the quick settings panel